reactive/k8s.py

```diff
@@ -28,15 +30,18 @@ def master(etcd):
     This actually runs the kubelet, which in turn runs a pod that contains the
     other master components.
     '''
-    render_manifest(etcd)
+    render_files(etcd)
     status_set('maintenance', 'Starting the kubernetes master container')
-    with chdir('dockerfiles/hyperkube'):
-        cmd = "docker-compose up -d"
-        check_call(split(cmd))
-    set_state('kubernetes-master.available')
-    set_state('proxy.available')
+    with chdir('files/kubernetes'):
+        # Start the kubelet container that starts the three master services.
+        check_call(split('docker-compose run -d kubelet'))
+        set_state('kubelet.available')
+        # Start the proxy container
+        status_set('maintenance', 'Starting the kubernetes proxy container')
+        check_call(split('docker-compose run -d proxy'))
+        set_state('proxy.available')
     # TODO: verify with juju action-do kubernetes/0 get-credentials
-    status_set('active', 'Kubernetes is started. verify with: kubectl get pods')
+    status_set('active', 'Kubernetes started')
```

Comment:
Why remove the action to verify? I think thats a good pattern to adopt - signaling to the user what to do at this phase... as in - "Go use it"